src/protocols/ssh/ssh.c

```diff
@@ -130,26 +130,36 @@ static guac_common_ssh_user* guac_ssh_get_user(guac_client* client) {
 
     } /* end if key given */
 
-    /* Otherwise, use password */
-    else {
-
-        /* Get password if not provided */
-        if (settings->password == NULL)
-            settings->password = guac_terminal_prompt(ssh_client->term,
-                    "Password: ", false);
-
-        /* Set provided password */
-        guac_common_ssh_user_set_password(user, settings->password);
-
-    }
-
     /* Clear screen of any prompts */
     guac_terminal_printf(ssh_client->term, "\x1B[H\x1B[J");
 
     return user;
 
 }
 
+/**
+ * A call-back function used to gather additional credentials from a client
+ * during a connection.  It takes the guac_client object and a string to
+ * display to the user, and returns the credentials entered by the user.
+ * 
+ * @param client
+ *     The guac_client object associated with the current connection
+ *     where additional credentials are required.
+ * 
+ * @param credName
+ *     The prompt text to display to the screen when prompting for the
+ *     additional credentials.
+ * 
+ * @return 
+ *     The string of credentials gathered from the user.
+ */
+char* guac_ssh_get_credential(guac_client *client, char* credName) {
```

Comment:
See above - the C convention for variables is `lowercase_with_underscores`.

Reply:
Changed, here, too.